SA3/qa_web/templates/qa_web/edit.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title> edit questions </title>
+	{% load static %}
+    <script src="{% static 'js/Markdown.Converter.min.js' %}"></script>
+    <script src="{% static 'js/Markdown.Editor.min.js' %}"></script>
+    <script src="{% static 'js/Markdown.Sanitizer.min.js' %}"></script>
+
+    <link rel="stylesheet" href="{% static 'css/pagedown.css' %}" />
+    {% include "qa_web/editor_style.html" %}
+</head>
+<body>
+    <a href="../"><button>Discard Changes</button></a>
+    <form method="post">
+      {% csrf_token %}
+      <input id="title" name = "title" type="text" placeholder="question title" value="{{ currentQ.content }}" required/>
```

Comment:
Here the preset value of the title field should be the title. So in this case I believe you would put `{{ currentQ.title }}`.